app/components/userAssignment/workWeekInput.tsx

```diff
@@ -80,6 +80,8 @@ export const WorkWeekInput = ({
 			);
 			setUserList(updatedUserList);
 			setProjectList(updatedProjectList)
+			refetchProjectList();
+			refetchUserList();
```

Comment:
I don't think this is a good solution but I was unsure how to get the archive button to swap to a delete button without the refetch. Changing the field canBeDeleted is not enough for react to rerender as it is too deep in object array to notice that it has changed.

Reply:
@zhukovdigital that seems to have fixed it 👍  thank you

Reply:
@HankC138 it seems like it`s working good now, good job!